src/components/Sidebar.js

```diff
@@ -1,13 +1,14 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import { SidebarContent, SidebarItem } from './Sidebar.style';
-
+import ThemeContext from '../context/ThemeContext';
 const Sidebar = ({ showMenu, onSelectCategory, categories }) => {
+  const ThemeCtx = useContext(ThemeContext);
```

Comment:
Aquí podrías destructurarlo para solo sacar la propiedad que te interesa en este componente.
```suggestion
  const { showMenu }  = useContext(ThemeContext);
```